pkg/build/scanner.go

```diff
@@ -56,43 +52,11 @@ func Scanner(manifest model.Manifest, githubToken string) error {
 
 	// Call imagescanner passing in base image name. If request times out, retry the request
 	baseImageName := "istio/base:" + baseVersion
-	numberRetries := 4
-	var resp *http.Response
-	for numberRetries > 0 {
-		resp, err = http.Get("http://imagescanner.cloud.ibm.com/scan?image=" + strings.TrimSpace(baseImageName))
-		if err != nil {
-			if err, ok := err.(net.Error); ok && err.Timeout() {
-				fmt.Println("Scanner request timed out. Need to run request again.")
-			} else {
-				return err
-			}
-		}
-		numberRetries--
+	cmd := util.VerboseCommand("trivy", "--ignore-unfixed", "--exit-code", "2", baseImageName)
```

Comment:
One issue is that privy is only part of the `master` build-tools at his point. If cherry-picking to the other releases, we need to add it there as well.